src/modules/previewpane/SvgThumbnailProvider/SvgThumbnailProvider.cs

```diff
@@ -143,7 +143,7 @@ public Bitmap GetThumbnailImpl(uint cx)
                 thumbnailDone.Set();
             };
 
-            var webView2Options = new CoreWebView2EnvironmentOptions("--block-new-web-contents --disable-features=RendererAppContainer");
+            var webView2Options = new CoreWebView2EnvironmentOptions("--block-new-web-contents");
```

Comment:
Naive here why doesn’t Monaco’s web view have this flag

Reply:
`--block-new-web-contents` was to deal with a possible security issue when svg files can contain javascript tags that run when we try to render them and open new pages. On Monaco the javascript that runs is not part of the files that are previewed so it doesn't suffer from the same vulnerability.